R/dot-net-wrapper.R

```diff
@@ -6,113 +6,47 @@
 #' sim <- loadSimulation(system.file("extdata", "Aciclovir.pkml", package = "ospsuite"))
 #'
 #' # looking at a reference to `.NET` simulation object
-#' sim$ref
+#' sim$pointer
```

Comment:
@PavelBal Why did we rename this to pointer? 
Where is the NetObject class coming from?

Reply:
`NetObject` is implemented in `{rSharp}`: https://www.open-systems-pharmacology.org/rSharp/articles/user-guide.html#object-pointers-and-the-netobject-class

In `rClr`, `ref` was an S3 object. Now, it is the external pointer `externalptr` that is held by `NetObject`.